docs/user-guide/security-features.md

```diff
@@ -1,20 +1,112 @@
 # Security Features
 
-{% hint style="info" %}
-### Prerequisite
+## Introduction
 
-Install any one of the following integrations for scanning vulnerabilities:
-* [Clair](../user-guide/integrations/clair.md)
-* Trivy
-{% endhint %}
+Devtron provides [DevSecOps](https://devtron.ai/product/devsecops) capabilities across your software development life cycle for both: the default CI/CD solution by Devtron as well as your existing CI/CD Tools.
+
+One of the key components of DevSecOps is the detection of security risks. Currently, Devtron supports the following types of scanning:
+
+* Image Scan
+* Code Scan (not available for Helm apps) 
+* Kubernetes Manifest Scan
+
+![Figure 1: Security Scan Results](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/security-scan.jpg)
+
+You can integrate a scanning tool of your choice. By default, Devtron integrates with Trivy using which you can scan for the following issues:
+
+* Vulnerability
+* License Risks
+* Misconfigurations
+* Exposed Secrets
+
+---
+
+## Where to Initiate the Scan
+
+### After Code Commit
```

Comment:
This doesn't guarantee a scan post code commit, it does before building the image

Reply:
@abhibhaw, input for this data was from Nishant's webinar on DevSecOps. `When you commit the code, it's essential to scan it before building a container image` this has been written already. 

Can you please suggest what needs to be fixed here?

Reply:
~After code commit~ Before building the artifact

Reply:
Changed the section title